lib/elastic_record/relation/search_methods.rb

```diff
@@ -207,12 +207,16 @@ def build_query_and_filter(query, filters)
           Arelastic::Searches::Query.new query_and_filter
         end
 
-        def build_query(query)
-          if query.is_a?(String)
-            query = Arelastic::Queries::QueryString.new query
+        def build_search_query
+          if query_value.is_a?(String)
+            Arelastic::Queries::QueryString.new query_value
+          else
+            query_value
           end
+        end
 
-          query
+        def build_search_filter
+          build_filter(filter_values)
```

Comment:
Why not do the same thing here? `build_filter(filters)` can be directly replaced by `build_search_filter`. The old method doesn't seem to be used anywhere else.